yodeploy/virtualenv.py

```diff
@@ -118,12 +150,31 @@ def check_requirements(ve_dir):
         sys.exit(1)
 
 
-def relocateable_ve(ve_dir):
+def relocateable_ve(ve_dir, python_version):
+    # We can only do this for Python 2 under Python 2
```

Comment:
Ah, the word "this" is probably ambiguous. It's referring to the function's goal, not the workaround to achieve it.